src/Microsoft.TestPlatform.CoreUtilities/Helpers/EnvironmentHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
do we want to add more in CoreUtilities dll? can we move this to Testplatform.Utilities?

Reply:
CoreUtilities have basic utilities on BCL. TestPlatform.Utilities has OM + BCL Utilities.